### PR TITLE
10749 change sql read

### PR DESCRIPTION
**Pull request by piensaenpixel**
Fix https://github.com/CartoDB/cartodb/issues/10749

image

@nobuti could take a look?

cc/ @carlostallon

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/components/infobox/infobox-factory.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = {
       mainAction: {
         label: opts.confirmLabel,
         type: opts.confirmType || 'link',
-        position: opts.confirmPosition || 'right'
+        position: opts.confirmPosition || 'left'
```
Comment (nobuti) [inline lib/assets/javascripts/cartodb3/components/infobox/infobox-factory.js]: this is going to change every infobox in the app, have you check everything is ok? with other infoboxes I mean...

File: app/assets/stylesheets/editor-3/infobox.css.scss [hunk 2]
```diff
@@ -14,7 +14,7 @@
   @include flex(2);
   margin-right: -25px;
   margin-left: -25px;
-  padding: 25px;
+  padding: 21px 25px 25px 25px;
```
Comment (nobuti) [inline app/assets/stylesheets/editor-3/infobox.css.scss]: Pleae

File: lib/assets/javascripts/cartodb3/components/infobox/infobox-factory.js (hunk 1 above)
Comment (piensaenpixel) [inline lib/assets/javascripts/cartodb3/components/infobox/infobox-factory.js]: true :) , fixed

**Comment by carlostallon** [comment]
the design looks good to me ;)

**Pull request merged**